src/ui-components/SwappableViewer/SwappableViewer.tsx

```diff
@@ -123,18 +123,17 @@ const SwappableViewer: React.FC<SwappableViewerProps> = ({
   };
 
   const updateViewport = (viewer: any) => {
-    const newIsZoomedIn = viewport.getMaxZoom() <= viewport.getZoom();
-    if (newIsZoomedIn !== isZoomedIn) {
-      setIsZoomedIn(newIsZoomedIn);
-    }
-
-    const newIsZoomedOut = viewport.getMinZoom() >= viewport.getZoom();
-    if (newIsZoomedOut !== isZoomedOut) {
-      if (!isZoomedOut && !regionFromAnnotations) {
-        viewport.resetView();
-      }
-      setIsZoomedOut(newIsZoomedOut);
-    }
+    // const newIsZoomedIn = viewport.getMaxZoom() <= viewport.getZoom();
+    // if (newIsZoomedIn !== isZoomedIn) {
+    //   setIsZoomedIn(newIsZoomedIn);
+    // }
+    // const newIsZoomedOut = viewport.getMinZoom() >= viewport.getZoom();
+    // if (newIsZoomedOut !== isZoomedOut) {
+    //   if (!isZoomedOut && !regionFromAnnotations) {
+    //     viewport.resetView();
+    //   }
+    //   setIsZoomedOut(newIsZoomedOut);
+    // }
```

Comment:
@stephenwf I've commented this out as it was causing a jumping effect whenever you try to zoom for me.

Reply:
I think the purpose of this code was to enforce a maximum and minimum zoom on the zoom buttons. I think without it you can continue to zoom in or out without it constraining the viewport. There may be another way to fix that particular issue. I believe this was functioning incorrectly when we had a "viewport-focus" as it was resetting to that when you zoomed too far out (zooming you back in).